src/js/controller/piskel/PublicPiskelController.js

```diff
@@ -17,7 +17,6 @@
     this.resetWrap_('setCurrentFrameIndex');
     this.resetWrap_('selectNextFrame');
     this.resetWrap_('selectPreviousFrame');
-    this.resetWrap_('setCurrentLayerIndex');
```

Comment:
That's a good point but then we need to review why selectLayer should keep firing a RESET event. It's really just a different flavour of setCurrentLayerIndex.

Not against doing that but I would prefer if it was isolated in its own commit.